gitlab/v4/objects/__init__.py

```diff
@@ -3880,11 +3880,22 @@ class ProjectPipelineVariableManager(ListMixin, RESTManager):
     _from_parent_attrs = {"project_id": "project_id", "pipeline_id": "id"}
 
 
+class ProjectPipelineTestReport(RESTObject):
+    _id_attr = None
+
+
+class ProjectPipelineTestReportManager(GetWithoutIdMixin, RESTManager):
+    _path = "/projects/%(project_id)s/pipelines/%(pipeline_id)s/test_report"
+    _obj_cls = ProjectPipelineTestReport
+    _from_parent_attrs = {"project_id": "project_id", "pipeline_id": "id"}
+
+
 class ProjectPipeline(RESTObject, RefreshMixin, ObjectDeleteMixin):
     _managers = (
         ("jobs", "ProjectPipelineJobManager"),
         ("bridges", "ProjectPipelineBridgeManager"),
         ("variables", "ProjectPipelineVariableManager"),
+        ("test_report", "ProjectPipelineTestReportManager")
```

Comment:
Hi @rmonat, thanks for this and sorry for the delay.

Could you please run `black` on your code so the linter is happy? See instructions in the readme: https://github.com/python-gitlab/python-gitlab#development-workflow

Then if you could just this to the docs maybe? Here's a recent example: https://github.com/python-gitlab/python-gitlab/commit/05cbdc224007e9dda10fc2f6f7d63c82cf36dec0.

bonus - If you'd like to also tackle adding unit tests, here is an example: https://github.com/python-gitlab/python-gitlab/blob/master/gitlab/tests/objects/test_pipeline_schedules.py.